app/components/CV/CV.tsx

```diff
@@ -0,0 +1,49 @@
+import { Experience } from './experience.js'
+import { LeftSideSections } from './leftSideSections.js'
+import { Section } from './section.js'
+import { CvDataType } from './types.js'
+
+const CV = (data: CvDataType) => {
+  return (
```

Comment:
**suggestion**: Consider using a more descriptive class name for the main container to improve readability and maintainability.